GamingAnthems/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "TechLewi",
+    "id": "590575157718941708"
+  },
+  "service": "GamingAnthems",
+  "description": {
+    "en": "No1 Gaming Radio Station"
+  },
+  "url": [
+    "gaminganthems.com",
+    "raidstop.co.uk",
+    "raidstop.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/9PEupja.png",
+  "thumbnail": "https://i.imgur.com/j7u9H9X.png",
+  "color": "#000000",

```

Comment:
Use a different color, the site offers other colors besides black. It adds more flare.

Reply:
Not required